src/main/resources/org/jenkinsci/plugins/workflow/flow/GlobalDefaultFlowDurabilityLevel/global.jelly

```diff
@@ -0,0 +1,17 @@
+<?jelly escape-by-default='true'?>
+<j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
+    <f:section title="Pipeline Speed/Durability Settings">
+        <f:entry title="${%Pipeline Default Speed/Durability Level}" field="durabilityHint">
+            <select class="setting-input" name="durabilityHint">
+                <f:option value="null" selected="${instance[field] == null}">None: use pipeline default (<j:out value="${descriptor.suggestedDurabilityHint.name()}" />)</f:option>
```

Comment:
Shouldn't the "None: use pipeline default" be in a messages.properties or whatever file?

Reply:
Theoretically yes, but something like 90% of pipeline doesn't do this, so it's hardly worthwhile to add it in just this one single place.